Userland/Applications/Spreadsheet/ConditionalFormatting.h

```diff
@@ -22,6 +22,11 @@ struct ConditionalFormat : public Format {
     String condition;
 };
 
+enum FormatType {
```

Comment:
```suggestion
enum class FormatType {
```

Let's not dump random names into the global ns

Reply:
Also, this can be squashed into the next commit, no need for it to be a separate commit.

Reply:
I read the code 1000 times to make sure everything was ok, and then forgot a keyword.